src/visualizers/VisualizerGrid.ts

```diff
@@ -319,7 +319,7 @@ class VisualizerGrid extends VisualizerDefault {
     }
 
     setup(): void {
-        this.sketch.createCanvas(400, 400)
+        this.sketch.createCanvas(this.sketch.width, this.sketch.height)
     }
```

Comment:
Individual visualizers do not need to create a canvas, as far as I know. None of the others do. If that's the only thing you have to do in the `setup()` function, you can just eliminate it. But make sure your visualizer re-draws properly in the main display after being shown in the bundle card, and re-draws properly when you go back and forth from the draw view to the bundle card view. My recollection is that if some things are not reset in setup, then the drawing doesn't come out right when you do those things. If you have any problem like that, take a look at what other visualizers do in setup and see if you need to emulate it.

Reply:
@gwhitney, this should be resolved in https://github.com/oliviaLovesTheSun/frontscope/commit/7ad0ad2a0c9d7d5f75701a97de50e3d18f681735. I tested this change manually by:

1. Creating a bundle using default params and random integers in range 0-9.
2. Ensuring visualizer was drawn on the initial "bundle card" screen.
3. Ensuring the visualizer was drawn on the "big" screen.
4. Clicking back to the "bundle card" screen and ensuring the visualizer was drawn.